app/pages/nics/Class4ExemptionCategoryPage.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nics
+
+import models.common.{BusinessId, TaxYear}
+import models.database.UserAnswers
+import pages.OneQuestionPage
+import play.api.mvc.Call
+
+case object Class4ExemptionCategoryPage extends OneQuestionPage[Boolean] {
+  override def toString: String = "class4ExemptionCategory"
+
+  override def cyaPage(taxYear: TaxYear, businessId: BusinessId): Call = ???
+
+  override def nextPageInNormalMode(userAnswers: UserAnswers, businessId: BusinessId, taxYear: TaxYear): Call = cyaPage(taxYear, businessId)
+
+  override def hasAllFurtherAnswers(businessId: BusinessId, userAnswers: UserAnswers): Boolean =
+    userAnswers.get(this, businessId).isDefined
```

Comment:
Similar to above, would specify that hasAllFurtherAnswers is dependent on this page being answered and the next page's hasAllFurtherAnswers = true (or you can leave a todo with the ticket number of the content ticket for this page.

Can also override val dependentPagesWhenNo and list the two pages afterthis that are skipped if you answer 'No'. That will ensure that the data for those pages is deleted if the user comes back and answers No.